server/build/webpack.js

```diff
@@ -33,12 +34,28 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
 
   const nodeModulesDir = join(__dirname, '..', '..', '..', 'node_modules')
 
+  const alias = {
+    'babel-runtime': babelRuntimePath
+  }
+
+  // Alias next's API in the next dev mode.
+  // This is only required when developing Next.js
+  if (process.env.NEXT_DEV) {
```

Comment:
I think this is required. This is basically to load modules from the root directory of `next`, not from app.